### DISCUSSION TITLE
So continuing on from a discussion from Vishnu and Nithin (On ARM64) (Support Needed)

### DISCUSSION THREAD

**Discussion by Alauraa** [chatwoot/chatwoot#4134] [Self hosted] [answered]
Getting a error after i copy the files like Gemfile and Gemfile.lock and package.json and yarn.lock, They have said that it is agnostic and should work on ARM, so i tried docker building the image myself on ARM, and im getting this error on step 17.

https://www.toptal.com/developers/hastebin/eyevevaxup.pl

And then it proceeds to fail.

**Comment by sojan-official** [comment]
@Alauraa  Could you specify the machine spec you are using? It might require around 4GB  RAM to compile the front end assets.

**Reply by sojan-official** [reply]
@vishnu-narayanan  could this be the case? https://github.com/freqtrade/frequi/issues/289

**Reply by sojan-official** [reply]
ref: https://stackoverflow.com/a/70546863/939299

**Comment by Alauraa** [comment]
> @Alauraa Could you specify the machine spec you are using? It might require around 4GB RAM to compile the front end assets.

4 OCPU's and 24GB's of Ram. Running on the Oracle Cloud Service.

**Reply by sirtonhao** [reply]
Docker não funciona em processadores ARM Oracle, Docker dont working on arm processor from oracle.

**Reply by Alauraa** [reply]
> Docker não funciona em processadores ARM Oracle, Docker dont working on arm processor from oracle.

Actually docker works perfectly fine on the Oracle Arm Processor.

**Reply by sojan-official** [reply]
@Alauraa  it should be a dependency issue. ref: https://stackoverflow.com/a/70546863/939299 
We will look into adding the packages into our images

**Comment by sirtonhao** [comment]
Docker dont working in Oracle ARM Processor.

**Comment by Alauraa** [comment]
> Docker dont working in Oracle ARM Processor.

Actually docker works perfectly fine on the Oracle Arm Processor.

**Reply by Lodimup** [reply]
Can confirm I have containers working in OCI

**Comment by vishnu-narayanan** [comment, marked as answer]
@Alauraa Docker builds now work on ARM64/apple silicon(m1). Could you please try now?

**Reply by paulm17** [reply]
Is there an docker build for M1 now?  Thanks

**Reply by Alauraa** [reply]
> @Alauraa Docker builds now work on ARM64/apple silicon(m1). Could you please try now?

So i tested it, but it still doesn't work tho there isn't really a documentation on the https://www.chatwoot.com/docs/self-hosted/deployment/docker that specifies a installation setup for ARM.

**Reply by Scared8062** [reply]
> So i tested it, but it still doesn't work tho there isn't really a documentation on the https://www.chatwoot.com/docs/self-hosted/deployment/docker that specifies a installation setup for ARM.

I don't understand. You said that it works perfectly fine on OCI Ampere Server, right?

**Reply by Scared8062** [reply]
> @Alauraa Docker builds now work on ARM64/apple silicon(m1). Could you please try now?

I visited the Chatwoot Docker Hub, there are no `arm64` tags. Should I install the `amd64` tag instead?
